index.bs

```diff
@@ -17,7 +17,7 @@ Complain About: accidental-2119 yes, missing-example-ids yes
 Favicon: logo-font-enumeration.svg
 </pre>
 <!--
```

Comment:
Move this out of the comment into the previous \<pre> block and it should show up in the spec header